### PR TITLE
Merge release 2.3 into trunk

### PR DESCRIPTION
**Pull request by mokagio**

### CODE REVIEW

**Review by jleandroperez — APPROVED** [review]
Thank you Gio!!

:shipit: sir!!

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,13 +2,13 @@ version: 2.1
 
 orbs:
   # This uses the Orbs located at https://github.com/wordpress-mobile/circleci-orbs
-  ios: wordpress-mobile/ios@0.0.27
+  ios: wordpress-mobile/ios@1.0
```
Comment (AliSoftware) [inline .circleci/config.yml:5]: 👍

File: Simplenote/AboutViewController.swift [hunk 2]
```diff
@@ -1,68 +1,118 @@
...
-//  Simplenote
-//
-
 import Cocoa
-import AutomatticTracks
+
 
 class AboutViewController: NSViewController {
-    struct Constants {
+    private enum Constants {
```
Comment (AliSoftware) [inline Simplenote/AboutViewController.swift:5]: 👌

File: Simplenote/BackgroundView.swift [hunk 3]
```diff
@@ -68,6 +68,10 @@ class BackgroundView: NSView {
         }
     }
 
+    /// Mouse Cursor that should be applied when the mouse hovers over
```
Comment (AliSoftware) [inline Simplenote/BackgroundView.swift:71]: > when the mouse hovers over

https://www.youtube.com/watch?v=y3oiPbNfw98 😂

**Review by AliSoftware — APPROVED** [review]
:shipit:

File: Simplenote/BackgroundView.swift (hunk 3 above)
Comment (mokagio) [reply Simplenote/BackgroundView.swift:71]: 🤣

Comment (jleandroperez) [reply Simplenote/BackgroundView.swift:71]: LOOOL

**Pull request merged**